readme.md

```diff
@@ -53,6 +53,16 @@ The following dependencies need to be installed on your system:
 			* MSVC v142 - VS 2019 C++ ARM64 build tools
 			* C++ ATL for v142 build tools (ARM64)
 
+### Note
+
+When you show the NVDA\extras\.vsconfig file in the source folder of NVDA to Visual Studio Istaller by following the steps of:
```

Comment:
Please move this to the Installed Dependencies section, after change the item that starts with "When installing Visual Studio" to just "When installing Visual Studio, you can import the config:"

Then add steps to import the configuration.

Please wrap the paths with back ticks EG:
```
`NVDA\extras\.vsconfig`
```

Please keep the steps explaining where the options are located.